.codeclimate.yml

```diff
@@ -27,26 +27,19 @@ checks:
   similar-code:
     config:
       threshold: # language-specific defaults. an override will affect all languages.
-    exclude_patterns:
-      - "simple_history/tests/"
-      - "simple_history/registry_tests/"
   identical-code:
     config:
       threshold: # language-specific defaults. an override will affect all languages.
-    exclude_patterns:
```

Comment:
https://docs.codeclimate.com/docs/advanced-configuration#section-exclude-patterns exclusion for codeclimate checks can only be made at global level, so I'm excluding tests